test/integration/util/util.go

```diff
@@ -26,27 +26,41 @@ import (
 	"time"
 
 	v1 "k8s.io/api/core/v1"
+	policy "k8s.io/api/policy/v1"
```

Comment:
I've taken the liberty to migrate policy pdb from V1beta1 to V1, there were 3 places using it